### PR TITLE
obsidian: Add version 0.9.4

### PR DESCRIPTION
**Pull request by nightscape**

### CODE REVIEW

File: bucket/obsidian.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "version": "0.9.4",
+    "description": "Obsidian is a powerful knowledge base that works on top of a local folder of plain text Markdown files.",
```
Comment (Ash258) [inline bucket/obsidian.json:3]:
```suggestion
    "description": "Powerful knowledge base that works on top of a local folder of plain text Markdown files.",
```

File: bucket/obsidian.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "homepage": "https://obsidian.md",
+    "license": "https://obsidian.md/eula",
```
Comment (Ash258) [inline bucket/obsidian.json:5]:
```suggestion
    "license": {
        "identifier": "Proprietary",
        "url": "https://obsidian.md/eula"
    },
```

File: bucket/obsidian.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            "hash": "db30e56d822792856605e398487fb807cdd54877829dbe7d9cd70f30a6907346",
+            "installer": {
+                "script": [
+                    "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                    "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall*\" -Force -Recurse"
+                ]
+            }
+        }
+    },
+    "bin": "Obsidian.exe",
```
Comment (Ash258) [inline bucket/obsidian.json:18]: What are available parameters?

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/obsidian.json (hunk 3 above)
Comment (nightscape) [reply bucket/obsidian.json:18]: I have tried to call it with `-h`, `--help` and `/?`. It always just starts the UI, so I'm assuming it doesn't have any.

Comment (Ash258) [reply bucket/obsidian.json:21]: Then remove the bin property

Comment (nightscape) [reply bucket/obsidian.json:21]: I thought that is there for allowing to call it from the command line.
But probably few people will do that.

**Review by Ash258 — APPROVED** [review]

**Pull request merged**